src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/EventsRepository.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Models;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Repository
+{
+    /// <summary>
+    /// Handles events repository. Notice that the all methods should modify the Subject attribute of the
+    /// CloudEvent, since cosmosDb fails if Subject contains slashes '/'.
+    /// </summary>
+    public class EventsRepository : IEventsRepository
+    {
+        private readonly ILogger _logger;
+        private readonly Uri _databaseUri;
+        private readonly Uri _collectionUri;
+        private readonly string _collectionId = "events";
+        private readonly string _partitionKey = "/subject";
+        private readonly string _triggerId = "trgUpdateItemTimestamp";
+        private static DocumentClient _client;
```

Comment:
This does not need to be static. The application will never make more than one EventsRepository because we are adding it to DI as a Singleton. See Startup.cs. This should also resolve a code smell reported by SonarCloud.
```suggestion
        private readonly DocumentClient _client;
```

Reply:
Done :)